### ISSUE TITLE
Column 'body' cannot be null

### ISSUE THREAD

**Issue by Chapullin** [bradtraversy/symphart#2]
Leaving empty body in the form, it returns:
`SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'body' cannot be null`

I edit \src\Entity\Article.php
```
/**
 * @ORM\Column(type="text", nullable=true)
 */
private $body = null;
```
and works perfectly.

**Comment by MoatazAbdAlmageed** [comment]
@Chapullin Good catch , I suggest you create a pull request for this update and ask @bradtraversy to check and merge it.
